README.md

```diff
@@ -1,2 +1,20 @@
-# RBMK-Model
-An OpenMC model of the RBMK core
+## Folders
+The **Model** folder contains the RBMK core model in a jupyter notebook file
+The **Analysis** folder will contain the analysis to be performed on the core
+
+## Documentation
+The geometry of this model is based off of dimensions from a paper looking into the design features and reasons for accident by Mikhail V. Malko  
+https://www.rri.kyoto-u.ac.jp/NSRG/reports/kr79/kr79pdf/Malko1.pdf  
+The core lattice arrangement comes from the paper A simplified analysis of the Chernobyl Accident by Bertrand Mercier https://www.sciencedirect.com/org/science/article/pii/S2491929221000078
+
+## Instructions
+The model should be able to be run as long as the user has installed openmc and it is running properly. The model is a jupyter notebook file and it is intended to be run in jupyter notebook. The model currently produces the component and core geometries.
+
+## Directory Structure
+```
+License
+README.md
+|-- Model/
+|   |--RBMK_Lattice.ipynb
+|-- Analysis/
+```
```

Comment:
This is a great structure! I think some of the content in "folders" would be easily readable by people who want to run your model if it's next to this content. How about merging the "folders" and "directory structures" into one section.